.gitlab-ci.yml

```
@@ -538,6 +538,33 @@ test-suite:base:dev:
     expire_in: 2 week
   allow_failure: true
 
+test-suite:edge+coverage:
+  stage: build
+  script:
+    - ulimit -s 16384
+    - ./configure -prefix $(pwd)/_build/install/coverage -native-compiler no
+    - make dunestrap
+    - dune build --instrument-with bisect_ppx --workspace dev/dune-workspace.bisect @runtest
+    - find _build/coverage/ -name '*.coverage' -size 0 -exec rm {} \; # workaround truncated coverage file bug
+    - bisect-ppx-report html --source-path=_build/coverage --coverage-path=_build/coverage -o _build/coq-coverage --title="Coq's Test suite"
+    - bisect-ppx-report coveralls coverage.json --source-path=_build/coverage --coverage-path=_build/coverage --git --service-name="github" --service-pull-request="${CI_COMMIT_BRANCH#pr-}" --service-job-id="$CI_JOB_ID"
```

I think we need to do
```suggestion
    - bisect-ppx-report Coveralls coverage.json --source-path=_build/coverage --coverage-path=_build/coverage --git --service-name="github" --service-pull-request="${CI_COMMIT_BRANCH#pr-}" --service-job-id="$CI_JOB_ID"
```
?